Masonry/NSArray+MASAdditions.h

```diff
@@ -42,6 +47,26 @@
  *
  *  @return Array of created/updated MASConstraints
  */
-- (NSArray *)mas_remakeConstraints:(void(^)(MASConstraintMaker *make))block;
+- (NSArray *)mas_remakeConstraints:(void (^)(MASConstraintMaker *make))block;
+
+/**
+ *  distribute with fixed spaceing
+ *
+ *  @param axisType     horizon/vertical
+ *  @param paddingSpace space
+ *  @param leadSpacing  head
+ *  @param tailSpacing  tail
+ */
+- (void)mas_distributeViewsAlongAxis:(MASAxisType)axisType withFixedSpacing:(CGFloat)paddingSpace leadSpacing:(CGFloat)leadSpacing tailSpacing:(CGFloat)tailSpacing;
```

Comment:
Rename `paddingSpace` to `fixedSpacing` and update the apple doc entry to reflect